core/src/main/scala/com/avast/cloud/metrics/datadog/api/package.scala

```diff
@@ -0,0 +1,8 @@
+package com.avast.cloud.metrics.datadog
+
+package object api {
+  type Tag = String
```

Comment:
consider something like https://github.com/estatico/scala-newtype

Reply:
i did - but i don't think it's worth it